wolox-training/ap-swift
# Component suggestions

## Summary ##
**Trello card**: https://trello.com/c/cVFzfaNb/20-componente-de-sugeridos

Suggestions Component

## Test Plan ##

  * Open the app
  * Login with Google
  * The main tab bar view should appear after a successfull login.
  * Press on a Book cell
  * The Book Detail  view should appear with a Suggestions Cell
  * Book images should appear on the Suggestion Component

## Screenshots ##

### iPhone 7
![component_suggestions](https://cloud.githubusercontent.com/assets/25587882/24049370/8c974548-0b0a-11e7-9d4b-2cd6213aafbb.png)

## Screencast ##
![component_suggestions](https://cloud.githubusercontent.com/assets/25587882/24049390/97fdc6c8-0b0a-11e7-8d25-ea596a96db96.gif)
